models/base.py

```diff
@@ -7,3 +7,15 @@
 
 class SMBase(BaseModel):
     """Base object for all models"""
+
+
+def parse_sql_bool(val: str | int | bytes) -> bool | None:
```

Comment:
Would not be better to have this func in db/python/utils or at least inside model/utils?

Reply:
Yep, good call!